### ISSUE TITLE
Insufficient Color Contrast error reported when using .show-brick

### ISSUE THREAD

**Issue by jrchamp** [ncstate/utility-bar#66]
Example: https://wave.webaim.org/report#/https://wolfware.ncsu.edu/help/

The problem appears to be that the text color is being set to transparent (i.e. rgb(0, 0, 0, 0)):
https://github.com/ncstate/utility-bar/blob/0421d700af0f790a5885c575ae27844abc599a29/src/main.scss#L624

The best idea I could come up with is to wrap the text in a span (within the anchor tag) and use "visually-hidden"/"sr-only" styles instead of transparency so that the automatic checkers are not throwing a false-positive on each of the thousands of pages that are using the utility-bar.
